content/source/docs/enterprise/api/variables.html.md

```diff
@@ -13,19 +13,23 @@ This set of APIs covers create, update, list and delete operations on variables.
 
 ## Create a Variable
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | /vars |
+`POST /vars`
 
-### Parameters
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload. (Properties without a default value are required.)
+
+Key path                    | Type   | Default | Description
+----------------------------|--------|---------|------------
+`data.type`                 | string |         | Must be `"vars"`.
+`data.attributes.key`       | string |         | The name of the variable.
+`data.attributes.value`     | string |         | The value of the variable.
+`data.attributes.category`  | string |         | Whether this is a Terraform or environment variable. Valid values are `"terraform"` or `"env"`.
+`data.attributes.hcl`       | bool   | `false` | Whether to evaluate the value of the variable as a string of HCL code. Has no effect for environment variables.
```

Comment:
I found an ambiguity when formatting... is my revision true, or does something else happen if you try to set hcl for an env var?